crates/oxc_linter/src/snapshots/no_export.snap

```diff
@@ -0,0 +1,46 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_export
+---
+  ⚠ eslint(jest/no-export): Do not export from a test file.
+   ╭─[no_export.tsx:1:1]
+ 1 │ export const myThing = 'invalid'; test('a test', () => { expect(1).toBe(1);});
+   · ▲
```

Comment:
Seems like the span is broken? Do you want to try and fix the caller? I think it is here 

https://github.com/web-infra-dev/oxc/blob/8867f9ada2ccc134a8910abcb9d95c960fef7189/crates/oxc_semantic/src/module_record/builder.rs#L350

https://github.com/web-infra-dev/oxc/blob/8867f9ada2ccc134a8910abcb9d95c960fef7189/crates/oxc_semantic/src/module_record/builder.rs#L317

Reply:
Done, is current position good?